.github/workflows/release.yaml

```diff
@@ -3,7 +3,7 @@ name: Release Charts
 on:
   push:
     branches:
-      - master
+      - "release-*"
```

Comment:
Should it be `release-1.19`?

Reply:
The reason for making this a pattern is so that we don’t need to update on every release.

Reply:
ack